airflow_dags/dags/train_model.py

```diff
@@ -0,0 +1,34 @@
+from airflow import DAG
+from airflow.operators.docker_operator import DockerOperator
+from airflow.operators.dummy_operator import DummyOperator
+from airflow.contrib.sensors.file_sensor import FileSensor
+from variables import *
```

Comment:
Ай-ай-ай, не стоит импортить *, т.к. любому читателю кода будет в итоге не понятно, что именно и откуда приехало.
Представьте ситуацию:
```
from module1 import *
from module2 import *
```

Страшно? Мне вот страшно. Лучше так:
```suggestion
from variables import (
    default_args,
    PROD_MODEL_DIR,
    VOLUME,
    FULL_PROD_MODEL_PATH,
    DATE_START,
    MODEL_DIR,
    DATA_DIR,
    PREDICTION_DIR,
)
```